allure-support/src/main/kotlin/com/kaspersky/components/alluresupport/files/resources/impl/DefaultAllureResourcesRootDirsProvider.kt

```diff
@@ -0,0 +1,12 @@
+package com.kaspersky.components.alluresupport.files.resources.impl
+
+import com.kaspersky.components.alluresupport.files.resources.AllureResourcesRootDirsProvider
+import java.io.File
+
+class DefaultAllureResourcesRootDirsProvider : AllureResourcesRootDirsProvider {
+    override val logcatRootDir = File("logcat")
+    override val screenshotsRootDir = File("screenshots")
+    override val videoRootDir = File("video")
+    override val viewHierarchy = File("view_hierarchy")
+    override val allureRootDir = File("allure-results")
```

Comment:
My concern is that this dir can be changed by the user.

Reply:
`io.qameta.allure.kotlin.util.PropertiesUtils` is public, you can use it in `allure-support` module. Seems like easy to support this.

Reply:
Yes, sorry, I'm an idiot. Looks like I've tried to use PropertiesUtils outside of allure support module